labs/NET_TESTING/sites/site_1/group_vars/SITE1_FABRIC.yml

```diff
@@ -0,0 +1,87 @@
+---
+fabric_name: SITE1_FABRIC
+
+# Set Design Type to L3L2 with EVPN VXLAN
+design:
+  type: l3ls-evpn
+
+# Spine Switches
+spine:
+  defaults:
+    platform: cEOS
+    loopback_ipv4_pool: 10.250.1.0/24
+    bgp_as: 65100
+  nodes:
+    - name: s1-spine1
+      id: 1
+      mgmt_ip: 192.168.0.10/24
+    - name: s1-spine2
+      id: 2
+      mgmt_ip: 192.168.0.11/24
+
+# Leaf Switches
+l3leaf:
+  defaults:
+    platform: cEOS 
+    spanning_tree_priority: 4096 
+    spanning_tree_mode: mstp
+    loopback_ipv4_pool: 10.250.1.0/24
+    loopback_ipv4_offset: 2
+    vtep_loopback_ipv4_pool: 10.255.1.0/24
+    uplink_switches: [ s1-spine1, s1-spine2 ]
+    uplink_interfaces: [ Ethernet2, Ethernet3 ]
+    uplink_ipv4_pool: 172.16.1.0/24
+    mlag_interfaces: [ Ethernet1, Ethernet6 ]
+    mlag_peer_ipv4_pool: 10.251.1.0/24
+    mlag_peer_l3_ipv4_pool: 10.252.1.0/24
+    virtual_router_mac_address: 00:1c:73:00:00:99
+  node_groups:
+    - group: S1_RACK1
+      bgp_as: 65101
+      nodes:
+        - name: s1-leaf1
+          id: 1
+          mgmt_ip: 192.168.0.12/24
+          uplink_switch_interfaces: [ Ethernet2, Ethernet2 ]
+        - name: s1-leaf2
+          id: 2
+          mgmt_ip: 192.168.0.13/24
+          uplink_switch_interfaces: [ Ethernet3, Ethernet3 ]
+    - group: S1_RACK2
+      bgp_as: 65102
+      nodes:
+        - name: s1-leaf3
+          id: 3
+          mgmt_ip: 192.168.0.14/24
+          uplink_switch_interfaces: [ Ethernet4, Ethernet4 ]
+        - name: s1-leaf4
+          id: 4
+          mgmt_ip: 192.168.0.15/24
+          uplink_switch_interfaces: [ Ethernet5, Ethernet5 ]
+    - group: S1_BRDR
+      bgp_as: 65103
+      evpn_gateway:
+        evpn_l2:
+          enabled: true
+        evpn_l3:
+          enabled: true
+          inter_domain: true
+      nodes:
+        - name: s1-brdr1
+          id: 5
+          mgmt_ip: 192.168.0.100/24
+          uplink_switch_interfaces: [ Ethernet6, Ethernet6 ]
```

Comment:
```suggestion
          uplink_switch_interfaces: [ Ethernet7, Ethernet7 ]
```

Reply:
Students will fix this as part of the ANTA lab guide :)